ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -747,21 +747,98 @@ end revSaveAsMobileDeployer
 
 ################################################################################
 
+local sCFBundleIcons
 private command revCopyMobileNewIcons pSettings, pBaseFolder, pAppBundle   
    revStandaloneProgress "Copying icons..."    
-   repeat for each line tType in ("iphone icon" & return & "retina icon,-114" & return & "iOS 7 retina icon,-120" & return & "iphone 6 plus icon,-180" & return & "iphone X icon,-180" & return & \
-         "ipad icon,-72" & return & "ipad retina icon,-144" & return & "iOS 7 ipad icon,-76" & return & "iOS 7 ipad retina icon,-152" & return & "iPad Pro 12.9 icon,-167")
+   /*
+   The entry for the AppStore icon in the JSON file should have the form:
+   {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x",
+      "filename" : "iTunesArtwork@2x.png"
+    }
+   */
+   local sIconsList 
+   put  \
+         "appstore icon,ios-marketing,1024x1024,1x,ItunesArtwork@2x.png" & return & \
```

Comment:
Is this meant to be scale `1x` but an `@2x.png` image file?

Reply:
Yes, although it seems strange, this is intended. This is the exact entry that is created if you use Xcode's interface to create the asset catalog.